javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -975,3 +974,134 @@ module ClassNode {
     }
   }
 }
+
+/**
+ * A data flow node that performs a partial function application.
```

Comment:
```suggestion
 * A data-flow node that performs a partial function application.
```

Reply:
Are you sure? None of the other classes use that convention, but I can update all of them if you like.

Reply:
Yeah, we're not being particularly consistent about this. I think the version with the hyphen is correct, but feel free to ignore.